raiden/network/proxies/channel_manager.py

```diff
@@ -67,11 +68,20 @@ def __init__(
         self.gasprice = gasprice
         self.poll_timeout = poll_timeout
 
-    def token_address(self):
+    def token_address(self) -> bytes:
```

Comment:
I prefer a specific type for the addresses, e.g `address = bytes`, bytes is not that meaningful.